src/recaptcha_native.ts

```diff
@@ -1,3 +1,17 @@
+// Copyright 2023 Google LLC
```

Comment:
@andrejpavlovic, can you change the year here for 2024, please?

Reply:
done